src/snek5000/make.py

```diff
@@ -255,3 +275,61 @@ def build(self, config):
                 return self.exec(*self.targets, config=config, force_incomplete=True)
             else:
                 return True
+
+
+def snek_make():
+    """Used for the command snek-make"""
+    parser = argparse.ArgumentParser(
+        prog="snek-make",
+        description="Execute Snakemake rules",
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+
+    parser.add_argument(
+        "rule",
+        nargs="?",
+        default=None,
+        help="Snakemake rule to be executed.",
+    )
+    parser.add_argument(
+        "-l",
+        "--list-rules",
+        action="store_true",
+        help="List rules and exit.",
+    )
+    parser.add_argument(
+        "-d",
+        "--dry-run",
+        action="store_true",
+        help="Dry run snakemake rules without executing.",
+    )
+    parser.add_argument(
+        "-np",
+        "--nproc",
+        type=int,
+        default=None,
+        help="Number of MPI processes.",
+    )
+    parser.add_argument(
+        "--clean-after-fail",
+        action="store_true",
+        help="Keep incomplete output files of failed jobs.",
```

Comment:
Should have been `Clean incomplete ...`?

Reply:
Done in https://github.com/snek5000/snek5000/pull/209/commits/c7c1369fab7438e84547d2ebdc52cbe9a7a80431